src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+
+    protected String color;
```

Comment:
```suggestion
    private String color;
```
Також додай гетери, сетери і конструктор

Reply:
Спасибо, выяснили на QnA, геттер добавил, сеттер не нужен. Конструктор создал.